src/common/withYearValidation.js

```diff
@@ -0,0 +1,69 @@
+import React from 'react'
+import { Redirect } from 'react-router-dom'
+
+/**
+ * Reconstructs the URL to point to the targetYear
+ * @param {Object} props Includes props from React Router
+ * @param {String} targetYear Latest available year for Publication/product
+ * @returns
+ */
+export const formatPath = (props, targetYear) => {
```

Comment:
I wonder if this function is generic enough that it can be helpful elsewhere; if it is, I don't think this is the place for it since we probably won't remember to associate this function within this file name.
if the function is more specific to the hoc, I don't think we need to export it.

Reply:
Looking at the path formatting in other sections of the app, they each require their own specific formatting rules. So this will stay local. I've removed the `export`.